src/themes/default/components/core/blocks/Form/BaseInput.vue

```diff
@@ -71,6 +73,16 @@ export default {
       type: [String, Number],
       default: ''
     },
+    min: {
```

Comment:
i guess it's also redundant because of the same reason. We don't need any internal validation for inpuits since they are validated outside with vuelidate

Reply:
if i remove this user will be able to add below 0 with buttons, this prevents that